Change note format (#2792)
theforeman/foreman-documentation: guides/common/modules/proc_using-an-imagebuilder-image-for-provisioning.adoc

When you configure {Project} for image provisioning, Anaconda installer partitions disks, downloads and mounts the image and copies files over to a host.
The preferred image type is TAR.

> [NOTE]
> ====
> The blueprint to build the TAR image must always include a kernel package.
> ====

Reviewer: ```suggestion
Ensure that your blueprint to build the TAR image includes a kernel package.
```

NOTE IMHO not really necessary.

Author: +1
